engine/select_members.py

```diff
@@ -10,172 +10,181 @@
 import logging
 import os
 import sys
-from collections import Counter
 from datetime import datetime
 
 import click
-import numpy as np
 
 from engine.tolerance import tolerance
 from util.click_util import cli_help
-from util.dataframe_ops import test_stats_file_with_tolerances
+from util.dataframe_ops import check_stats_file_with_tolerances
 from util.log_handler import logger
 
 
-# finds members and a corresponding tolerance factor validating for all stats files.
 def find_members_and_factor_validating_for_all_stats_files(
     random_tolerance_file_name,
```

Comment:
The word "random" is still used at many places in this script, however the new method is no longer random. Maybe one more thing to change?

Reply:
Yes, you are absolutely right.